test/test_faiss_and_milvus.py

```diff
@@ -192,6 +193,21 @@ def test_finding(document_store, retriever):
     assert len(prediction.get('documents', [])) == 1
 
 
+@pytest.mark.slow
+@pytest.mark.parametrize("retriever", ["dpr"], indirect=True)
+@pytest.mark.parametrize("document_store", ["faiss", "milvus"], indirect=True)
+@pytest.mark.parametrize("batch_size", [4, 6])
+def test_delete_docs_with_filters(document_store, retriever, batch_size):
+    document_store.write_documents(DOCUMENTS)
+    document_store.update_embeddings(retriever=retriever, batch_size=batch_size)
+
+    document_store.delete_documents(filters={"name": ["name_1", "name_2", "name_3", "name_4"]})
+
+    documents = document_store.get_all_documents()
+    assert len(documents) == 2
+    assert {doc.meta["name"] for doc in documents} == {"name_5", "name_6"}
+
```

Comment:
Should we also check here how many vectors are stored in FAISS to make sure that the delete worked there? I think that's would be an interesting addition as `test_delete_documents_with_filters()` already covers the "SQL side" without embeddings.